src/php/ext/grpc/channel.c

```diff
@@ -192,8 +174,8 @@ PHP_METHOD(Channel, __construct) {
     }
     efree(args.args);
   }
-  channel->target = ecalloc(override_len + 1, sizeof(char));
-  memcpy(channel->target, override, override_len);
+  channel->target = ecalloc(target_length + 1, sizeof(char));
```

Comment:
Now that it is no longer being passed to calls, do you still need to store `channel->target`? If not, you can also remove that field from the struct in `channel.h`.